automation/src/test/resources/bucket-tests/hail_samples.csv

```diff
@@ -0,0 +1,2505 @@
+,sample,age,ancestry,bmi,fg,fi,hdl,height,ldl,population,sex,tc,tg,whr
```

Comment:
This is synthetic data, and the same as in this workspace: https://app.terra.bio/#workspaces/fc-product-demo/2019_ASHG_Reproducible_GWAS

For some reason I couldn't reproduce the Hail error on a small (5x5) dataframe. But this data does reproduce it. Maybe the dataset needs to be larger in order to be distributed to the workers -- just guessing.